CRM/Contact/Form/Edit/TagsAndGroups.php

```diff
@@ -58,8 +58,8 @@ public static function buildQuickForm(
     $groupElementType = 'checkbox',
     $public = FALSE
   ) {
-    if (!isset($form->_tagGroup)) {
-      $form->_tagGroup = [];
+    if (!isset($tagGroup)) {
+      $tagGroup = [];
     }
```

Comment:
This condition will always be true, so can just be 
```suggestion
    $tagGroup = [];
```

Reply:
Oh yeah, I sshould have spotted that. Thanks.